FLEDGE.md

```diff
@@ -743,6 +743,66 @@ For detailed specification and explainers of the trusted key-value server, see a
 
 ##### 3.1.1 Cross-Origin Trusted Server Signals
 
+If the key-value server is on a different origin than the corresponding script, additional steps are
+needed to ensure that sensitive information is not inappropriately leaked to, or is manipulated by,
+a third party.
+
+In case of bidder signals, the following changes occur:
+1. The fetch is a cross-origin CORS fetch with `Origin:` set to the buyer script's origin.
+2. The value is passed to `crossOriginTrustedSignals` parameter, not the `trustedBiddingSignals`
+   parameter, and there is one more level of nesting denoting the server's origin, e.g:
+
+```
+{
+  'https://www.kv-server.example': {
+    'keys': {
+        'key1': arbitrary_json,
+        'key2': arbitrary_json,
+        ...},
+    'perInterestGroupData': {
+        'name1': {
+            'priorityVector': {
+                'signal1': number,
+                'signal2': number,
+                ...}
+        },
+        ...
+    }
+  }
+}
+```
+
+Seller signals have additional requirements, as the `trustedScoringSignalsURL` is provided by the
```

Comment:
This is true in the case where the publisher is invoking runAdAuction and is the top-level seller. But otherwise, sellers are providing & operating the TSS url.

Reply:
It's still in context of the publisher page, and thus not really having the full trust of the seller's origin though?